README.md

```diff
@@ -1 +1,61 @@
-# JavaScript-group-capstone
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# JavaScript-group-capstone
+This is JavaScript Module capstone project, and this project is about building a web application based on an external API. we have selected an API that provides data and another wich adds the comments and likes about movie series and then built the webapp around it. 
+
+## Built With
+
+• HTML, CSS 
+• JavaScript
+• VSCode
+• Webpack
+. ES6
+. API Services
+
+## Demo Video
+[Demonstration Video](https://drive.google.com/drive/folders/1wtzX7u8tSCPgpJryMD9TiMrA8NoiLf1s?usp=sharing)
+
+## Movie screen shot
+<img width="788" alt="moviescreenshot" src="web.png">
+
+## Page Link 
+Clone and run NPM install locally then NPM run start
+http://localhost:8080/
```

Comment:
- I'd suggest you add a live deploy link to your page in place of the above information, so that non technical personnel can access your page without having to clone the repo 👍🏽